docs/ppp/iteration-5-16112022.md

```diff
@@ -0,0 +1,32 @@
+### Plans 📆
+
+- Create invoices
+- Send Payments
+- Bug fixing and UX improvements
+- Replacing mock data with real data
+- Show payment history
+- Containerize maker
+- Wiring up close (settle) CFD
+- Persisting CFDs
+
+### Progress ✅
+
+- Fund wallet screen
+- Setup maker service with ldk integration
+- Integrated our custom fork of rust-lightning supporting DLCs (custom outputs)
+- Wired up open CFD (still with mock amounts)
+- Improved logging
+- Started ldk node on the phone
+- Open channel with maker
+- Fetch bid, ask and index prices from bitmex
+- Show history of on-chain transactions
+
+### Problems ✋
+
+- None, after a busy week, we are on good track. Still a lot to be done, but only doing from here on.
```

Comment:
Shall we be honest about our short cuts and the outcome of our LDK extension? 

- When implementing our custom protocol on the LDK which allows us to add DLCs to the channel we accepted one short coming: while a DLC is added, you cannot accept or send a payment. This would break the DLC and would make the outputs unspendable. For the tournament we decided to accept this shortcoming but we will focus on this afterwards.

Reply:
I am going to add that.